assets/javascripts/search.config.js

```diff
@@ -25,35 +25,30 @@ $(window).ready(function() {
   // Tracking the search results.
   //
   var trackAnalytics = function(data, query) {
-    var total = data.total;
-    if (total > 0) {
-      _gaq.push(['_trackEvent', 'search', 'with results', query, total]);
-    } else {
-      _gaq.push(['_trackEvent', 'search', 'not found', query, 0]);
-    }
+    // NOOP
```

Comment:
Isn't it better to just completely get rid of these? Empty branching just seems like a pain :) 
Or there are too many references to bother with ?